src/test/java/org/gridsuite/loadflow/server/LoadFlowParametersTest.java

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.loadflow.server;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.gridsuite.loadflow.utils.assertions.Assertions.*;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import org.gridsuite.loadflow.server.dto.parameters.LoadFlowParametersInfos;
+import org.gridsuite.loadflow.server.entities.parameters.LoadFlowParametersEntity;
+import org.gridsuite.loadflow.server.repositories.parameters.LoadFlowParametersRepository;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.http.MediaType;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.powsybl.loadflow.LoadFlowParameters;
+
+/**
+ * @author Ayoub LABIDI <ayoub.labidi at rte-france.com>
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+@Transactional
+public class LoadFlowParametersTest {
+
+    private static final String URI_PARAMETERS_BASE = "/v1/parameters";
+
+    private static final String URI_PARAMETERS_GET_PUT = URI_PARAMETERS_BASE + "/";
+
+    @Autowired
+    protected MockMvc mockMvc;
+
+    @Autowired
+    protected ObjectMapper mapper;
+
+    @Autowired
+    private LoadFlowParametersRepository parametersRepository;
```

Comment:
junit and spring boot aren't affected by visibility of autowired
```suggestion
    @Autowired
    MockMvc mockMvc;

    @Autowired
    ObjectMapper mapper;

    @Autowired
    LoadFlowParametersRepository parametersRepository;
```